docs/release-info/release-notes.md

```diff
@@ -65,55 +184,234 @@ This section contains detailed release notes separated by areas.
 
+- You can now assign tiered replications to tiers that aren't currently online [#18050](https://github.com/apache/druid/pull/18050)
+- You can now filter tasks by the error in the Task view [#18057](https://github.com/apache/druid/pull/18057)
+- Improved SQL autocomplete and added JSON autocomplete [#18126](https://github.com/apache/druid/pull/18126)
+- Updated the web console to use the Overlord APIs instead of Coordinator APIs when managing segments, such as marking them as unused [#18172](https://github.com/apache/druid/pull/18172)
+
 ### Ingestion
 
+- Improved concurrency for batch and streaming ingestion tasks [#17828](https://github.com/apache/druid/pull/17828)
+- Removed the `useMaxMemoryEstimates` config. When set to false, Druid used a much more accurate memory estimate that was introduced in Druid 0.23.0. That more accurate method is the only available method now. The config has defaulted to false for several releases [#17936](https://github.com/apache/druid/pull/17936)
+
 #### SQL-based ingestion
 
 ##### Other SQL-based ingestion improvements
 
 #### Streaming ingestion
 
+##### Multi-stream supervisors (experimental)
+
+You can now use more than one supervisor to ingest data into the same datasource. Use the `id` field to distinguish between supervisors ingesting into the same datasource (identified by `spec.dataSchema.dataSource` for streaming supervisors).
+
+When using this feature, make sure you set `useConcurrentLocks` to `true` for the `context` field in the supervisor spec.
+
+[#18149](https://github.com/apache/druid/pull/18149) [#18082](https://github.com/apache/druid/pull/18082)
+
+##### Supervisors and the underlying input stream
+
+Seekable stream supervisors (Kafka, Kinesis, and Rabbit) can no longer be updated to ingest from a different input stream (such as a topic for Kafka). Since such a change is not fully supported by the underlying system, a request to make such a change will result in a 400 error. 
+
+[#17955](https://github.com/apache/druid/pull/17955) [#17975](https://github.com/apache/druid/pull/17975)
+
 ##### Other streaming ingestion improvements
 
+- Improved streaming ingestion so that it automatically determine the maximum number of columns to merge [#17917](https://github.com/apache/druid/pull/17917)
+
+
 ### Querying
 
+#### Metadata query for segments
+
+You can use a segment metadata query to find the list of projections attached to a segment.
+
+[#18119](https://github.com/apache/druid/pull/18119)
+
+
 #### Other querying improvements
 
+- You can now perform big decimal aggregations using the MSQ task engine [#18164](https://github.com/apache/druid/pull/18164)
+- Changed `MV_OVERLAP` and `MV_CONTAINS` functions now aligns more closely with the native `inType` filter [#18084](https://github.com/apache/druid/pull/18084)
+- Improved query handling when some segments are missing on Historicals. Druid no longer incorrectly returns partial results [#18025](https://github.com/apache/druid/pull/18025)
```

Comment:
Reworded since this is a bit more specific:
```suggestion
- Improved query handling when segments are temporarily missing on Historicals but not detected by Brokers. Druid doesn't return partial results incorrectly in such cases. [#18025](https://github.com/apache/druid/pull/18025)
```